automatic/inkscape/tools/chocolateybeforemodify.ps1

```diff
@@ -0,0 +1,7 @@
+﻿$app = Get-WmiObject -Class Win32_Product -Filter "Name Like 'Inkscape%'"
```

Comment:
Please don't use the `Get-WmiObject`, but rather use a combination of [Get-UninstallRegistryKey](https://chocolatey.org/docs/helpers-get-uninstall-registry-key) and [Uninstall-ChocolateyPackage](https://chocolatey.org/docs/helpers-uninstall-chocolatey-package) instead.

If you could update with that, I would be more than happy to merge this one in.

Reply:
also forgot, this must be done in the install script, not the before modify script (*Since the before modify script is run on both upgrade and uninstall, but doesn't fail the package if it doesn't work)*)

Reply:
@AdmiringWorm I tried that before but ran into problems. I've spent enough time on this so I encourage someone else to have a go at this.